pkgs/shells/zsh/default.nix

```diff
@@ -81,6 +82,9 @@ EOF
       ${lib.getBin buildPackages.zsh}/bin/zsh -c "zcompile $out/etc/zprofile"
     ''}
     mv $out/etc/zprofile $out/etc/zprofile_zwc_is_used
+
+    mkdir -p $out/share/doc/
+    mv $out/share/zsh/htmldoc $out/share/doc/zsh-$version
```

Comment:
What is the reason for including the $version in the path?

Reply:
This way you can build nice `buildEnv` with documentation of different versions.

Reply:
Ah. Makes sense!

Reply:
Thats a very specific usecase which is normally not required and other files would likely still collide. Please just keep the doc where the build system put it.

Reply:
Upstream build system puts them into completely wrong location, $prefix/share/zsh/htmlref.